docs/tutorial/notebook/train_ss_model_using_pytorch.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>Needs editing to remove repeating language:</p><p> </p><blockquote>Run inference with a pre-trained model. For this example, we will use the <code>RandLANet</code> model.</blockquote><p>Rephrase:</p><blockquote>Train a model. We will train a model using the <code>SemanticKITTI</code> dataset and <code>RandLANet</code> model.</blockquote><p>Rephrase for clarity:</p><blockquote>Run an inference and run a test. We will run an inference using the 'training' split that use a pointcloud and display a result. However, a test is run on a pre-defined test set rather than a pass pointcloud.</blockquote><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p>